docs/usage/configuration-options.md

```diff
@@ -1190,6 +1190,23 @@ A preset alternative to the above is:
 }
 ```
 
+To match specific ports you have to add a protocol to `matchHost`:
+
+```json
+{
+  "hostRules": [
+    {
+      "matchHost": "https://domain.com:9118",
+      "enabled": false
+    }
+  ]
+}
+```
+
+<!-- prettier-ignore -->
+!!! warning
+    Using `matchHost` without a protocol will lead to a behaviour as there is no `matchHost` configuration at all.
```

Comment:
```suggestion
    Using `matchHost` without a protocol behaves the same as if you had set no `matchHost` configuration.
```

I think this is closer to what you really mean? The current text needs work in any case.

Reply:
For me both sentences have the same meaning. :men-shrugging:

I have no strong opinion regarding this tough.